### ISSUE TITLE
Auto doc comment generation and other formatOnType features broken

### ISSUE THREAD

**Issue by CynicalBusiness** [dotnet/vscode-csharp#7184]
## Environment data
<details>
<summary>`dotnet --info` output:</summary>
<p>

```
.NET SDK:
 Version:           8.0.201
 Commit:            4c2d78f037
 Workload version:  8.0.200-manifests.3097af8b

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.22631
 OS Platform: Windows
 RID:         win-x64
 Base Path:   C:\Program Files\dotnet\sdk\8.0.201\
```

</p>
</details>

VS Code version: `1.89.1`

C# Extension version: `2.31.19` and `2.32.14`

## Steps to reproduce

1. Switch to `v2.31.19`, `2.32.14`, or later and `Restart Extensions`
2. Type `///` above any method, class, etc.
3. No further text appears

## Expected  behavior
Code snippets and other formatOnType behaviors work as they did in `2.30.28` (via `Switch to Release Version`).

## Actual behavior
Code snippets and other formatOnType behavior, such as automatically inserted spaces and `*` characters for multiline comments or XML tags for documentation comments, are not functioning.

## Additional context
I've changed no other settings or other extensions. The issue appears (that is, stops working) when switching to `2.31`+ and disappears (that is, works again) when going back to `2.30`.

If this is a new change (new setting, etc.) rather than a bug, I haven't been able to find any documentation on it.

Edit: Initially filed the report for preview, but I've re-tested with the latest release and it's still present. Updated to match.

**Comment by ChristoWolf** [comment]
Hi!

I experience the same issues using the same versions as @CynicalBusiness.

**Comment by artemkoloskov** [comment]
Same here on `2.31.19`

**Comment by CynicalBusiness** [comment]
Looks like the version went stable and since others are seeing it appear on this version, including myself on `v2.31.19`, I updated the title.

I can only imagine there must be some special case which triggers/breaks this behavior if it has gone unnoticed thus far?

**Comment by badeend** [comment]
Can confirm it's broken on `v2.31.19` and downgrading to `v2.30.28` "fixed" it.

**Comment by dibarbet** [comment]
Should be fixed in an upcoming prerelease, will update here when its out.

**Comment by dibarbet** [comment]
This is published in the 2.34.10 prerelease (will go to the release channel next week).
